tests/python/relay/test_pass_to_a_normal_form.py

```diff
@@ -198,12 +218,4 @@ def test_gradient_if():
 
 
 if __name__ == "__main__":
-    test_explicit_bound()
-    test_order()
-    test_if()
-    test_recursion()
-    test_ref()
-    test_let()
-    test_nat_add()
-    test_function()
-    test_gradient_if()
+    pytest.main([__file__])
```

Comment:
since you're there can you indulge my OCD and write:
    import sys
    import pytest

    sys.exit(pytest.main([__file__] + sys.argv[1:]))